libindy_vdr/src/ffi/pool.rs

```diff
@@ -102,7 +107,7 @@ fn handle_pool_refresh(
             cache.update(&init_txns, latest_txns)?;
         }
         if let Some(new_txns) = new_txns {
-            let runner = PoolBuilder::new(config, new_txns).node_weights(node_weights).refreshed(true).into_runner()?;
+            let runner = PoolBuilder::new(config, new_txns, None).node_weights(node_weights).refreshed(true).into_runner()?;
```

Comment:
I think this means that after a successful pool refresh (with new pool transactions), the new pool runner will no longer have a cache?